tests/framework/Optimizers/GeneticAlgorithms/discrete/unconstrained/testGAMaxwoRepTwoPointsCrossover.xml

```diff
@@ -0,0 +1,172 @@
+      <lowerBound>1</lowerBound>
+      <upperBound>6</upperBound>
+      <strategy>withoutReplacement</strategy>
+    </UniformDiscrete>
+  </Distributions>
+
+  <Optimizers>
+    <GeneticAlgorithm name="GAopt">
+      <samplerInit>
+        <limit>20</limit>
+        <initialSeed>42</initialSeed>
+        <writeSteps>every</writeSteps>
+        <type>max</type>
+      </samplerInit>
+
+      <GAparams>
+        <populationSize>20</populationSize>
+        <parentSelection>rouletteWheel</parentSelection>
+        <reproduction>
+          <crossover type="twoPointsCrossover">
+            <crossoverProb>0.8</crossoverProb>
+          </crossover>
+          <mutation type="swapMutator">
+            <mutationProb>0.9</mutationProb>
+          </mutation>
+        </reproduction>
+        <fitness type="invLinear">
+          <a>2.0</a>
+          <b>1.0</b>
+        </fitness>
+        <survivorSelection>fitnessBased</survivorSelection>
+      </GAparams>
+
+      <convergence>
+        <objective>-1</objective>
+      </convergence>
+
+      <variable name="x1">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+
+      <variable name="x2">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+
+      <variable name="x3">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+
+      <variable name="x4">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+
+      <variable name="x5">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+
+      <variable name="x6">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+
+      <objective>ans</objective>
+      <TargetEvaluation class="DataObjects" type="PointSet">optOut</TargetEvaluation>
+    <Sampler class="Samplers" type="MonteCarlo">MC_samp</Sampler>
+    </GeneticAlgorithm>
+  </Optimizers>
+
+  <Samplers>
+    <MonteCarlo name="MC_samp">
+      <samplerInit>
+        <limit>20</limit>
+        <initialSeed>20021986</initialSeed>
+      </samplerInit>
+      <variable name="x1">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+      <variable name="x2">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+      <variable name="x3">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+      <variable name="x4">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+      <variable name="x5">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+      <variable name="x6">
+        <distribution>uniform_dist_woRepl_1</distribution>
+      </variable>
+    </MonteCarlo>
+  </Samplers>
+
+    <Models>
+      <ExternalModel ModuleToLoad="../../../../../AnalyticModels/optimizing/myLocalSum.py" name="myLocalSum" subType="">
+        <variables>x1,x2,x3,x4,x5,x6,ans</variables>
```

Comment:
Use inputs and outputs instead of variables

Reply:
Fixed